### PR TITLE
Allow parsing of tweets without media (#3)

### PR DESCRIPTION
**Pull request by oneroyalace**
Checks tweet JSON for `media` attribute before processing it. Should fix broken tests in https://github.com/TechAndCheck/zenodotus/pull/44

### CODE REVIEW

File: lib/birdsong/tweet.rb [hunk 1]
```diff
@@ -49,9 +49,13 @@ def parse(json_tweet, includes)
       @language = json_tweet["lang"]
       @author_id = json_tweet["author_id"]
 
-      # A sanity check to make sure we have everything in there correctly
-      media_items = includes["media"].filter do |media_item|
-        json_tweet["attachments"]["media_keys"].include? media_item["media_key"]
+      # A sanity check to make sure we have media in there correctly
+      if includes.has_key? "media"
```
Comment (oneroyalace) [inline lib/birdsong/tweet.rb:53]: @cguess Not sure if this is idiomatic (Rubynic? Rubonic?)

Comment (cguess) [reply lib/birdsong/tweet.rb:53]: "The ruby way" or whatever.

File: test/tweet_test.rb [hunk 2]
```diff
@@ -34,6 +34,13 @@ def test_that_a_tweet_cant_be_found_works
     end
   end
 
+  def test_that_a_tweet_can_have_no_media
+    tweet = Birdsong::Tweet.lookup("20").first
+    assert_equal tweet.id, "20"
+    assert_not_nil tweet.image_file_names
+    assert_equal 0, tweet.image_file_names.count
```
Comment (cguess) [inline test/tweet_test.rb:41]: add checks for videos as well, since we want to make sure *no* media was collected.

**Review by cguess — APPROVED** [review]
Looks great, add one assertion and we're good!

**Pull request merged**